collections/sentinel-1-grd.yaml

```diff
@@ -163,19 +163,42 @@ Summaries:
     - HV
     - VV
     - VH
-  sar:bands:
+  raster:bands:
     - name: VV
       description: 'Single co-polarization, vertical transmit/vertical receive'
-      polarization: VV
+      openeo:gsd:
+        value:
+        - - 0.00009259259
+          - 0.00009259259
+        - - 0.0002572016
+          - 0.0002572016
+        - - 0.0003703704
+          - 0.0003703704
+        unit: °
     - name: VH
       description: 'Dual-band cross-polarization, vertical transmit/horizontal receive'
-      polarization: VH
+      openeo:gsd:
+        value:
+        - - 0.00009259259
+          - 0.00009259259
+        - - 0.0002572016
+          - 0.0002572016
```

Comment:
Let's use 1/3'' = 10m as reference basis here as well to not switch. So I'd go with "0.00023148147"

Reply:
Thanks, changed.
I also calculated the numbers with enough precision so that they are all equally long, same as in https://github.com/eurodatacube/public-collections/pull/160#discussion_r793523252

Reply:
Perfect, thanks!